### PR TITLE
Telescience

### PR DESCRIPTION
**Pull request by AsV9**
### Intent of your Pull Request

Readds telescience.

#### Changelog

:cl:  
rscadd: Telescience.
/:cl:

### CODE REVIEW

File: yogstation/code/game/machinery/computer/computer_circuitboards.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/item/circuitboard/computer/telesci_console
```
Comment (GrayRachnid) [inline yogstation/code/game/machinery/computer/computer_circuitboards.dm]: shouldn't this be modularized?

**Comment by GrayRachnid** [comment]
Shouldn't the whole thing be modularized?

**Comment by AsV9** [comment]
:thinking:

File: yogstation/code/modules/telesci/telepad.dm [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+	active_power_usage = 500
+	var/stage = FALSE
+
+/obj/machinery/telepad_cargo/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/wrench))
+		playsound(src, 'sound/items/Ratchet.ogg', 50, 1)
+		if(anchored)
+			anchored = FALSE
+			to_chat(user, "<span class='caution'>\The [src] can now be moved.</span>")
+		else if(!anchored)
```
Comment (Amelia0010) [inline yogstation/code/modules/telesci/telepad.dm]: just else.

**Pull request merged**